pms/tests/test_pms_folio_invoice.py

```diff
@@ -208,7 +202,6 @@ def test_invoice_partial_folio_diferent_partners(self):
 
     def test_invoice_partial_folio_wrong_qtys(self):
         # ARRANGE
```

Comment:
a short descriptive text of the task of the test can help a lot to understand what is being tested